deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -255,13 +171,17 @@ spec:
                 - name: ROOK_DISCOVER_DEVICES_INTERVAL
                   value: 60m
                 - name: ROOK_HOSTPATH_REQUIRES_PRIVILEGED
-                  value: "false"
+                  value: "true"
                 - name: ROOK_ENABLE_SELINUX_RELABELING
                   value: "true"
                 - name: ROOK_ENABLE_FSGROUP
                   value: "true"
-                - name: ROOK_DISABLE_DEVICE_HOTPLUG
+                - name: ROOK_ENABLE_FLEX_DRIVER
                   value: "false"
+                - name: ROOK_ENABLE_DISCOVERY_DAEMON
+                  value: "false"
+                - name: ROOK_DISABLE_DEVICE_HOTPLUG
```

Comment:
This should remain false.

Reply:
Same here. We don't need hotplug, as the block devices are provided via PVs we don't need hotplug.

Reply:
Agreed, we can keep this disabled since we don't need to trigger an orchestration for a new raw device. The orchestration would be triggered by the CR when the StorageClassDeviceSet(s) change